src/Export/Field/BaseFieldInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Omikron\FactFinder\Oxid\Export\Field;
+
+interface BaseFieldInterface
```

Comment:
I find this confusing to be honest. We have this empty interface and FieldInterface which inherits from it but in some places we have 
`use Omikron\FactFinder\Oxid\Export\Field\BaseFieldInterface as FieldInterface`
i think alias name should not be equal to the class or interface name when what we are aliasing is actually extending it.

What is the benefit of introducing BaseFieldInterface?

Reply:
Exporter use field classes (like Omikron\FactFinder\Oxid\Export\Field\Brand), which are implements FieldInterface, I thought that maybe in future we will need similar mechanism for field of Categories or any other export types. So I decide to create common interface for all existing and possible field types.

Reply:
Why FieldInterface could not be such a common Interface?

Reply:
yeah, in general you're right, let me change it